fixtures/sample-esy/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sample-esy",
+  "version": "0.1.0",
+  "description": "Starter Reason Esy Pesy Project",
+  "esy": {
+    "build": "dune build -p sample-esy"
+  },
+  "scripts": {
+    "start": "esy x SampleEsyApp.exe",
+    "test": "esy x RunSampleEsyTests.exe"
+  },
+  "dependencies": {
+    "@opam/dune": "*",
+    "@esy-ocaml/reason": "*",
+    "ocaml": "4.6.x"
+  },
+  "devDependencies": {
+    "@opam/merlin": "*",
```

Comment:
is merlin really required? ocaml-lsp-server should include it.

Reply:
Not needed. But it shouldn't affect since it's a test project. Removing it so that tests run faster